building-pyramids/java/printingloops2.java

```diff
@@ -0,0 +1,16 @@
+public class printingloops2{
+    public static void main(String[] args){
+        int count = 1;
+        String dot = "*";
+        String space = "                                        ";
```

Comment:
Another way to initialize a string with a certain number of characters (let's say 40 in this case) is like this:
    char[] chars = new char[40];
    Arrays.fill(chars, ' ');
    String space = new String(chars);